packages/flutter_tools/doc/daemon.md

```diff
@@ -60,6 +60,19 @@ The `version()` command responds with a String with the protocol version.
 
 The `shutdown()` command will terminate the flutter daemon. It is not necessary to call this before shutting down the daemon; it is perfectly acceptable to just kill the daemon process.
 
+### daemon.getSupportedPlatforms
+
+The `getSupportedPlatforms()` command will enumerate all platforms supported by the project located at the provided
+`projectRoot`. It returns a List of strings which describe the set of all possibly supported platforms. Possible values
```

Comment:
Should we wrap the return in a map like the others, so if we ever need to add additional data to this it can be non-breaking?

Reply:
Done